src/components/policy/policy_regular/src/policy_manager_impl.cc

```diff
@@ -70,14 +70,17 @@ PolicyManagerImpl::PolicyManagerImpl()
           new AccessRemoteImpl(std::static_pointer_cast<CacheManager>(cache_)))
     , retry_sequence_timeout_(kDefaultRetryTimeoutInMSec)
     , retry_sequence_index_(0)
+    , count_application_(0)
     , timer_retry_sequence_(
           "Retry sequence timer",
           new timer::TimerTaskImpl<PolicyManagerImpl>(
               this, &PolicyManagerImpl::OnPTUIterationTimeout))
     , ignition_check(true)
     , retry_sequence_url_(0, 0, "")
     , send_on_update_sent_out_(false)
-    , trigger_ptu_(false) {}
+    , trigger_ptu_(false)
+    , ptu_requested_(false)
+    , last_registered_app_id_("") {}
```

Comment:
is it `policy_app_id`? if so, then i think that the variable's name should be changed accordingly

Reply:
@mked-luxoft  done [fe7b393](https://github.com/smartdevicelink/sdl_core/pull/3208/commits/fe7b393fe80f89ac9137d6dd9b41a0dfd6ba28c6)